src/components/ApplicationContainer/index.tsx

```diff
@@ -20,25 +24,19 @@ const ApplicationContainer = ({ children, problemDetail }) => {
   }, [errorToDisplay]);
 
   return (
-    <>
-      {/* <Header
-        skipNav={{ target: "main-content" }}
-        navData={navConfig.current}
-      /> */}
-      <div className="nypl-library-card-app nypl-ds">
-        <main id="main-content" className="main main--with-sidebar">
-          <div className="content-header">
-            <Banner />
-          </div>
-          <div className="content-primary content-primary--with-sidebar-right">
-            <ApiErrors ref={errorSection} problemDetail={errorToDisplay} />
-            {children}
-          </div>
-          <div className="content-secondary content-secondary--with-sidebar-right" />
-        </main>
-      </div>
-      <Footer />
-    </>
+    <TemplateAppContainer
+      className="nypl-library-card-app"
+      breakout={<Banner />}
+      contentPrimary={
+        <>
+          <ApiErrors ref={errorSection} problemDetail={errorToDisplay} />
+          {children}
+        </>
+      }
+      sidebar="right"
+      contentSidebar={<></>}
```

Comment:
@bigfishdesign13 check out this oddity (?). The width isn't full-width and doing this does help add the empty right sidebar layout that is currently in production.

Reply:
And what happens if you don’t add that fragment?

Reply:
You get the full width layout.

Reply:
Does the sidebar DOM element exist?  Or is it omitted altogether?

Reply:
Actually, if I remove this line, it still renders with an empty right sidebar. Just having `sidebar="right"` is enough apparently.